packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -277,6 +281,22 @@ class WebView extends StatefulWidget {
   /// [WebViewController.evaluateJavascript] can assume this.
   final PageFinishedCallback onPageFinished;
 
+  /// Invoked when a web resource has failed to load.
+  ///
+  /// This can be called for any resource (iframe, image, etc.), not just for
+  /// the main page.
+  ///
+  /// On Android, the error code will be a constant from
+  /// [WebViewClient](https://developer.android.com/reference/android/webkit/WebViewClient#summary).
+  /// (e.g. The error code of `WebViewClient.ERROR_AUTHENTICATION` on Android
+  /// translates to `AUTHENTICATION` in Dart.
+  ///
+  /// On iOS, the error code will represent the "domain" and the "code" for an
+  /// `NSError` in Objective-C. The format will be `$domain: $code`. See
```

Comment:
Is $code the http status code?

Reply:
I think it's possible for it to be an http status code, but the documentation doesn't say anything about it:
https://developer.apple.com/documentation/webkit/wknavigationdelegate/1455623-webview?language=objc

However, their https://developer.apple.com/documentation/webkit/wkerrorcode?language=objc states that these are possible values. I can't tell if these are the ONLY possible values or not.